modules/apps/headless-apio/media-object/media-object-apio-client-test/src/testIntegration/java/com/liferay/media/object/apio/client/test/MediaObjectApioTest.java

```diff
@@ -441,19 +394,11 @@ private String _getDocumentsHref(String folderName) {
 		);
 	}
 
-	private File _readFile(String fileName) {
-		Class<?> clazz = getClass();
-
-		URL url = clazz.getResource(fileName);
-
-		return new File(url.getFile());
-	}
-
+	private URL _contentSpaceHref;
```

Comment:
Can we use a String here instead?
If not, it should probably be named _contentSpaceHrefURL